src/plugins/imagery/mixins/imageryData.js

```diff
@@ -46,7 +46,6 @@ export default {
 
         // kickoff
         this.subscribe();
-        this.requestHistory();
```

Comment:
This is used by two components - `ImageryView.vue` and https://github.com/nasa/openmct/blob/master/src/plugins/imagery/components/ImageryTimeView.vue We need to make sure we are not introducing a regression in the time view here.

Reply:
Oh yeah need to test both views for realtime and historical data.

Reply:
Agreed, this change was actually already merged into master and isn't present in the release/2.0.4 branch. I had paired with Nikhil to troubleshoot the clear data imagery bug.

https://github.com/nasa/openmct/pull/5223/files

Reply:
Agreed, this change was actually already merged into master and isn't present in the release/2.0.4 branch. I had paired with Nikhil to troubleshoot the clear data imagery bug.

https://github.com/nasa/openmct/pull/5223/files